index.html

```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/4.png" alt="work 4" class="img4" />
+          </div>
+          <div class="work__description">
+            <h2>Multi-Post Stories</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
+          </div>
+        </article>
+      </section>
+      <section class="about-me" id="about">
+        <div class="about-me-container">
+          <div class="about-me__title">
+            <h2>About Myself</h2>
+          </div>
+          <div class="about-me__description">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a href="#" class="medias">let's connect</a>
+          <ul class="social-media">
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/twitter.svg"
+                  alt="twitter"
+                  title="follow me on twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/Linkedin_icon.svg"
+                  alt="linkedin"
+                  title="follow me on linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/medium.svg"
+                  alt="medium"
+                  title="follow me on"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="images/icons/github.svg"
+                  alt="github"
+                  title="follow me on github"
+              /></a>
+            </li>
```

Comment:
- [ ] Please ensure that this section matched the original design guidelines (margins, paddings, spacing etc). At the moment, the original design is centered while yours is not
- [ ] Also try and prevent the horizontal scrolling as this does not create a good user experience.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158475458-837a0818-6b36-464b-a5ee-f3b6f16d7e95.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158475539-74fa5bdc-19f5-4879-8c45-23bf7fc8a6c2.png)